src/styles/main.scss

```diff
@@ -0,0 +1,88 @@
+$width: 100px;
+$calendar-width: 712px;
+$gap: 1px;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+@each $day, $marginMultiplier in $days {
+  .square-#{$day} {
+    $baseSize: 100px;
+    $gap: 1px;
+    $size: $baseSize * $marginMultiplier + $gap;
+
+    width: $size;
+    height: $size;
+    margin: $gap;
+  }
+}
+
+body {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  flex-wrap: wrap;
+  gap: 1px;
+
+  width: $calendar-width;
+
+  display: flex;
+  padding: 0, 10px;
```

Comment:
The padding property is using a comma (`,`) instead of a space. It should be `padding: 0 10px;` to apply padding correctly.